### PR TITLE
nfd-master: explicit state variable for the node updater pool

### PR DESCRIPTION
**Pull request by marquiz**

### CODE REVIEW

File: pkg/nfd-master/updater-pool.go [hunk 1]
```diff
@@ -158,18 +154,14 @@ func (u *updaterPool) start(parallelism int) {
...
 	u.Lock()
 	defer u.Unlock()
 
-	if u.queue == nil || u.queue.ShuttingDown() {
-		klog.InfoS("the NFD master updater pool is not running.")
-		return
-	}
-
-	if u.nfgQueue == nil || u.nfgQueue.ShuttingDown() {
+	if !u.started {
```
Comment (TessaIO) [inline pkg/nfd-master/updater-pool.go:164]: why don't we use the u.running method?

Comment (marquiz) [reply pkg/nfd-master/updater-pool.go:164]: As that's also behind a lock

Comment (TessaIO) [reply pkg/nfd-master/updater-pool.go:164]: perfect thanks, didn't see the first lock 😄

**Pull request merged**